include/blmc_robots/blmc_joint_module.hpp

```diff
@@ -401,10 +408,22 @@ class BlmcJointModules
         }
     }
 
+    /**
+     * @brief Set the polarities of the joints
+     * 
+     * @param polarities 
+     */
+    void reverse_joint_polarities(std::array<bool, COUNT> polarities)
+    {
+        for(size_t i = 0; i < COUNT; i++)
+        {
```

Comment:
What was the agreement we found during the software discussion meeting? Should it be like this or `for (...) {`?

Reply:
We said that brackets should always comes next line.